main.js

```diff
@@ -0,0 +1,55 @@
+//Working with variables
+let user = "John Doe";
+console.log(user);
+const student = 'Mariya Tsimashkova';
+console.log(student);
+    user = student;
+//expected output: "Mariya Tsimashkova"
+console.log(user);
+
+
+//Working with primitives
+let test = 1;
+    test++;
+    test += '1';
+//expected output: "21"
+console.log(test);
+test -= 1;
+//expected output: "20"
+console.log(test);
+test = Boolean();
+//expected output: "false"
```

Comment:
Boolean(undefined) is "false", but Boolean("20") is "true". What variant is expected in the task?